input.css

```diff
@@ -14,8 +14,8 @@
   transform: scale(1.2, 1.2);
 }
 .mi-input[type=search] {
-  padding-left: 34px;
-  background: url("data:image/svg+xml,%3Csvg width='20' height='20' viewBox='0 0 20 20' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M19.5814 19.5815C19.8492 19.3136 19.9997 18.9503 19.9997 18.5715C19.9997 18.1927 19.8492 17.8294 19.5814 17.5615L15.7657 13.7458C15.7067 13.6881 15.6693 13.6119 15.66 13.5299C15.6506 13.4479 15.6698 13.3652 15.7143 13.2958C16.8861 11.5221 17.3577 9.37662 17.0376 7.27509C16.7175 5.17356 15.6285 3.26573 13.9818 1.92143C12.335 0.577128 10.2478 -0.107886 8.12477 -0.000823963C6.0017 0.106238 3.994 0.997751 2.49085 2.50089C0.987711 4.00404 0.0961981 6.01174 -0.0108642 8.13481C-0.117927 10.2579 0.567088 12.3451 1.91139 13.9918C3.25569 15.6385 5.16352 16.7275 7.26505 17.0476C9.36658 17.3677 11.5121 16.8962 13.2857 15.7243C13.3546 15.6791 13.4369 15.659 13.5188 15.6673C13.6008 15.6756 13.6774 15.7119 13.7357 15.77L17.5514 19.5858C17.6845 19.7196 17.8429 19.8257 18.0172 19.898C18.1916 19.9703 18.3786 20.0073 18.5673 20.0069C18.7561 20.0065 18.9429 19.9687 19.117 19.8957C19.291 19.8227 19.4489 19.7159 19.5814 19.5815ZM2.85715 8.57148C2.85715 7.4413 3.19229 6.3365 3.82018 5.39679C4.44807 4.45708 5.34052 3.72466 6.38467 3.29216C7.42882 2.85966 8.57777 2.7465 9.68624 2.96699C10.7947 3.18747 11.8129 3.73171 12.612 4.53086C13.4112 5.33002 13.9554 6.34821 14.1759 7.45667C14.3964 8.56514 14.2832 9.71409 13.8507 10.7582C13.4182 11.8024 12.6858 12.6948 11.7461 13.3227C10.8064 13.9506 9.70161 14.2858 8.57143 14.2858C7.05591 14.2858 5.60246 13.6837 4.53082 12.6121C3.45919 11.5404 2.85715 10.087 2.85715 8.57148Z' fill='%23AEB9CB'/%3E%3C/svg%3E%0A") no-repeat scroll 8px center, white;
+  padding-left: 32px;
+  background: url("data:image/svg+xml,%3Csvg width='20' height='20' viewBox='0 0 24 22' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M14.6267 8.43836C14.6267 11.8561 11.8561 14.6267 8.43836 14.6267C5.02062 14.6267 2.25 11.8561 2.25 8.43836C2.25 5.02062 5.02062 2.25 8.43836 2.25C11.8561 2.25 14.6267 5.02062 14.6267 8.43836Z' stroke='%23aeb9cb' stroke-width='2.5'/%3E%3Crect x='15.0979' y='14.1614' width='7.34678' height='1.32449' rx='0.662244' transform='rotate(45 15.0979 14.1614)' fill='%23aeb9cb' stroke='%23aeb9cb'/%3E%3Crect x='13.7847' y='13.2916' width='1.05276' height='0.697347' transform='rotate(45 13.7847 13.2916)' stroke='%23aeb9cb' stroke-width='0.697346'/%3E%3C/svg%3E%0A") no-repeat scroll 8px center, white;
```

Comment:
@ammapspeople Does it have an unintended effect that the viewbox defines an icon that is 22x24px? I'd expect it to be 24x24px.

Reply:
I actually changed it since the looking glass is placed in the top of the svg.

Reply:
@ammapspeople Let it be known that I do not approve of unevenly-sized icons in theory, but in practice, I'll allow it if it makes it look correctly 🙂